apis/v1beta1/shared_types.go

```diff
@@ -66,6 +66,7 @@ type ParentReference struct {
 	// Gateway has the AllowedRoutes field, and ReferenceGrant provides a
 	// generic way to enable any other kind of cross-namespace reference.
 	//
+	//<experimental:description>
```

Comment:
One tiny nit, probably best to stick with the `<gateway:experimental` prefix we're using elsewhere.
```suggestion
	// <gateway:experimental:description>
```